app/main.py

```diff
@@ -12,13 +15,18 @@
 
 @app.get("/games/{game_id}")
 async def read_game(game_id: int):
-    return {"game_id": game_id}
+    response = Games.get_item(Key={"Id": game_id})
+    print(response)
```

Comment:
You don't usually want to commit statements that are designed for debugging, so you'll probably either want to remove the statement before pushing a PR or consider using python's logging module for a more robust solution.

```
import logging

logger = logging.getLogger(__name__)

def read_game(…):
    …
    logger.debug("Got game: %s", response)
```